### ISSUE TITLE
Ipython console stops responding if large text is displayed

### ISSUE THREAD

**Issue by kparicharak** [spyder-ide/spyder#14610]
### What steps will reproduce the problem?

I use Spyder on Windows 10. In default setup IPython pane is at right bottom part (quandrant) the screen. When I enter command that has large text in output e.g. help on int (int?). It shows the output with three dots in the end (...) probably indicating there is more text than it can be displayed. But nothing happends after this point. Whatever I press, it does not respond. I tried pressing Enter to get to prompt or Ctrl-I. But it does not respond.

Steps
1. Make sure Ipython pane is occupying lower right quadrant
2. On iPython prompt, write "int?" and press enter
3. Try pressing Enter or another key to get to prompt

## Versions

* Spyder version: 4.2.0 
* Python version: 3.7.3
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree >=3.0.2            :  3.1.0 (OK)
IPython >=4.0                   :  7.19.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.7.2 (OK)
pygments >=2.0                  :  2.7.3 (OK)
pylint >=1.0                    :  2.6.0 (OK)
pyls >=0.36.1;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.1.1             :  0.3.0 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.1 (OK)
qtconsole >=4.7.7               :  4.7.7 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.4 (OK)
setuptools >=39.0.0             :  51.0.0.post20201207 (OK)
sphinx >=0.6.6                  :  3.4.3 (OK)
spyder_kernels >=1.10.0;<1.11.0 :  1.10.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog >=0.10.3               :  0.10.4 (OK)
zmq >=17                        :  20.0.0 (OK)

# Optional:
cython >=0.21                   :  None (NOK)
matplotlib >=2.0.0              :  3.3.2 (OK)
numpy >=1.7                     :  1.19.2 (OK)
pandas >=1.1.1                  :  1.1.1 (OK)
scipy >=0.17.0                  :  1.5.2 (OK)
sympy >=0.7.3                   :  None (NOK)
```

**Comment by steff456** [comment]
Hi @kparicharak,

When you open the docs in the IPython terminal you have to press `Q` to exit that view and get the terminal responsive once again. We enhanced the UI/UX of this feature in our latest release, if you are interested in it you can upgrade the version of Spyder in your system. 

Please let me know if this works for you.

**Comment by AmerM137** [comment]
Hi @steff456,

This may be related to this issue. The new label appears correctly when typing something "pd.DataFrame?" however I noticed that pgUP/pgDown keyboard shortcuts don't work anymore if the documentation requested spans more than one page.

From what I can tell, only mousewheel up/dn works. Keyboard shortcuts worked fine in 4.1.5.

**Comment by ccordoba12** [comment]
Hey @AmerM137, this problem should be fixed in our latest version available in Anaconda (5.3.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).
